docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+
+```yaml
+global:
+  identity:
+    auth:
+      type: "MICROSOFT"
+      issuer: "https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/v2.0"
+      issuerBackendUrl: "https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/v2.0"
+      tokenUrl: "https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/oauth2/v2.0/token"
+      jwksUrl: "https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/discovery/v2.0/keys"
+      identity:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://identity.camunda.example.com"
+        initialClaimName: "email"
+        initialClaimValue: test.user@camunda.com
+      optimize:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://optimize.camunda.example.com"
+      core:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        audience: "00000000-0000-0000-0000-000000000000"
+        tokenScope: "00000000-0000-0000-0000-000000000000/.default"
+        redirectUrl: "https://core.camunda.example.com"
+      console:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        wellKnown: https://login.microsoftonline.com/00000000-0000-0000-0000-000000000000/v2.0/.well-known/openid-configuration
+        audience: "00000000-0000-0000-0000-000000000000"
+        existingSecret: "password-string-literal"
+        redirectUrl: "https://console.camunda.example.com"
+      webModeler:
+        clientId: "00000000-0000-0000-0000-000000000000"
+        clientApiAudience: "00000000-0000-0000-0000-000000000000"
+        publicApiAudience: "00000000-0000-0000-0000-000000000000"
+        redirectUrl: "https://modeler.camunda.example.com"
+```
+
+If you would like some more guidance relating to authentication, then please refer to the following guides:
+
+- [Connect to an OpenID Connect provider](/self-managed/setup/guides/connect-to-an-oidc-provider.md)
+
+### Connect External Databases
+
+The next stage of the production setup is configuring databases. To make it easy for testing, the Camunda Helm Chart provides external, dependency Helm Charts for Databases such as Bitnami Elasticsearch Helm Chart and Bitnami PostgresQL Helm Chart. Within a production setting, these dependency charts should be disabled and production databases should be used instead. For example, instead of the Bitnami Elasticsearch dependency chart, we will use Amazon OpenSearch, and instead of the Bitnami PostgreSQL dependency chart, we will use Amazon Aurora PostgreSQL.
```

Comment:
Optional suggestion to link to e.g. elasticsearch and progresql bitnami charts via artifacthub.io
- https://artifacthub.io/packages/helm/bitnami/postgresql
- https://artifacthub.io/packages/helm/bitnami/elasticsearch